testing/tests/DevExpress.ui.widgets/gantt.tests.js

```diff
@@ -0,0 +1,34 @@
+import $ from "jquery";
+const { test } = QUnit;
+import "common.css!";
+import "ui/gantt";
+
+QUnit.testStart(() => {
+    const markup = '<div id="gantt"></div>';
+    $("#qunit-fixture").html(markup);
+});
+
+const TREELIST_SELECTOR = ".dx-treelist";
+
+const moduleConfig = {
+    beforeEach: () => {
+        const tasks = [
+            { "id": 1, "parentId": 0, "title": "Software Development", "start": new Date("2019-02-21T05:00:00.000Z"), "end": new Date("2019-07-04T12:00:00.000Z"), "progress": 31 },
+            { "id": 2, "parentId": 1, "title": "Scope", "start": new Date("2019-02-21T05:00:00.000Z"), "end": new Date("2019-02-26T09:00:00.000Z"), "progress": 60 },
+            { "id": 3, "parentId": 2, "title": "Determine project scope", "start": new Date("2019-02-21T05:00:00.000Z"), "end": new Date("2019-02-21T09:00:00.000Z"), "progress": 100 },
+            { "id": 4, "parentId": 2, "title": "Secure project sponsorship", "start": new Date("2019-02-21T10:00:00.000Z"), "end": new Date("2019-02-22T09:00:00.000Z"), "progress": 100 }
+        ];
+
+        this.$element = $("#gantt").dxGantt({
+            taskDataSource: tasks
+        });
+        this.instance = this.$element.dxGantt("instance");
+    }
+};
+
+QUnit.module("Gantt", moduleConfig, () => {
+    test("should render treeList", (assert) => {
+        const treeListElement = this.$element.find(TREELIST_SELECTOR);
+        assert.ok(treeListElement.length === 1);
+    });
+});
```

Comment:
Add tests for each Gantt option initialization and for each option changing

Reply:
Tests will be added after the optionsChanged method implementation

Reply:
Are you not using TDD? Tests about options initialization can be added now.